<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuiteReagentPopupForm</class>
 <widget class="QWidget" name="SuiteReagentPopupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSuiteReagent">
      <attribute name="title">
       <string>&amp;Наборы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="CTableView" name="tblSuiteReagent"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFind">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="chkOnlyByTest">
         <property name="text">
          <string>Только наборы относяшиеся к тесту</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkNotOverdue">
         <property name="text">
          <string>Не просрочены</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkStartOperation">
         <property name="text">
          <string>Введены в работу</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkNotOverLimit">
         <property name="text">
          <string>Не превышен плановое количество</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>123</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CApplyResetDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CApplyResetDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>library/DialogButtonBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
